### PR TITLE
[@types/google.analytics] Step to number + Added cookiePath for Google Analytics

### PR DESCRIPTION
**Pull request by FreshRob**
Step can be typenumber
 https://developers.google.com/analytics/devguides/collection/analyticsjs/enhanced-ecommerce

Added cookiePath  
https://www.optimizesmart.com/google-analytics-cookies-ultimate-guide/

Metrics can be type string | number 
https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets

### CODE REVIEW

**Comment by FreshRob** [comment]
@pckujawa could you review this please

File: types/google.analytics/index.d.ts [hunk 1]
```diff
@@ -524,7 +525,7 @@ declare namespace UniversalAnalytics {
         socialNetwork?: string;
         socialTarget?: string;
         some?: string;
-        step?: boolean;
+        step?: number;
```
Comment (patkujawa-wf) [inline types/google.analytics/index.d.ts]: For backwards-compatibility, should this be a union of number and boolean instead?

Comment (FreshRob) [inline types/google.analytics/index.d.ts]: You are right, I have made it either or. I have also now gone through and updated all metrics to also be either string or number based on https://developers.google.com/analytics/devguides/collection/analyticsjs/custom-dims-mets

**Comment by patkujawa-wf** [comment]
:thumbsup: nice improvement.

**Review by patkujawa-wf — APPROVED** [review]
:thumbsup:

**Pull request merged**